### DISCUSSION TITLE
Welcome, Drakonz!

### DISCUSSION THREAD

**Discussion by DrakonZZZ** [ParanoidUser/yolo#434] [Subscribe]
Trying to get achievements on my profile :)

**Comment by ParanoidUser** [comment]
Ahoy @DrakonZZZ, welcome aboard!

The invitation to join the collaborator group has been sent to you. Thanks for your interest in this project!
Feel free to post your own questions to the community, or share your path to earning other badges. 😊 Let me know if you have any questions.

Cheers!

**Comment by DrakonZZZ** [comment]
Thank you

**Reply by ParanoidUser** [reply]
Here is a few tips that might be helpful:
 - https://github.com/ParanoidUser/yolo/discussions/18
 - https://github.com/ParanoidUser/yolo/discussions/26
 - For `pull-shark`, same as `YOLO`, but instead a reviewer has to merge the PR (instead of creator).
 - Final tip: If you can complete the PR update within 5 minutes, it will open `Quickdraw`.
